src/model/resources/storage/storage_all.jl

```diff
@@ -10,6 +10,7 @@ function storage_all!(EP::Model, inputs::Dict, setup::Dict)
 	dfGen = inputs["dfGen"]
 	Reserves = setup["Reserves"]
 	CapacityReserveMargin = setup["CapacityReserveMargin"]
+	scale_factor = setup["ParameterScale"] == 1 ? ModelScalingFactor : 1
```

Comment:
It might be nice to think about loading this value in some other function, perhaps during load-time. In general, ModelScalingFactor and setup["ParameterScale"] is only referenced during the load_* and write_* phases, not during model construction. (The sole exception is in the hydrogen electrolyzer code.)

Reply:
Perhaps this could be added to the inputs[] dictionary? @lbonaldo . Or added to each relevant resource, once that's done.

Reply:
Yes, I think that would simplify the usage of the scaling factor. Something like:
```
inputs["scale_factor"] = setup["ParameterScale"] == 1 ? ModelScalingFactor : 1
```
in `load_inputs`, and then inside the functions we get the value from the `inputs` `Dict`?
```
scale_factor = inputs["scale_factor"]
```

Reply:
That's a good idea! I was actually thinking of constructing the `VirtualChargeDischargeCost` at load time and sticking *that* in `inputs`!